L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigisPostLS1_cfi.py

```diff
@@ -1,5 +1,10 @@
 import FWCore.ParameterSet.Config as cms
 
+print " ######################################################################################################### "
+print " # WARNING: the module L1Trigger.CSCTriggerPrimitives.cscTriggerPrimitiveDigisPostLS1_cfi is deprecated. # "
```

Comment:
I think that all files including it in the release should be cleaned up, at least the standard configurations/sequences.
We have enough noise already during configuration building.